lua/nvim-navic/init.lua

```diff
@@ -317,6 +317,7 @@ function M.get_data()
 			type = lsp_num_to_str[v.kind],
 			name = v.name,
 			icon = config.icons[v.kind],
+			scope = vim.deepcopy(v.scope)
```

Comment:
Is deepcopy necessary?

Reply:
@SmiteshP 

It is not needed, I have updated this.
I was under the impression that without deepcopy it might be possible to tamper with internal state.